mabl snippets/native mobile testing/mobileZoomInAtScreenCenter.js

```diff
@@ -0,0 +1,71 @@
+function mablJavaScriptStep(mablInputs, callback) {
+  async function callAppium() {
+    const driver = await mabl.mobile.getDriver();
+    // Find the element by xpath
+    const target = await driver.findElement("xpath", "//*");
+    // Get the rect of the element
+    const center = await getElementCenterXY(target);
+    // Define the actions to perform
+    const actions = [
+      {
+        type: "pointer",
+        id: "leftFinger",
+        parameters: { pointerType: "touch" },
+        actions: [
+          // Move the pointer to the left of the center
+          { type: "pointerMove", duration: 0, x: center.x * 0.45, y: center.y },
```

Comment:
Is this what you intended? The `center.x` should already be the center of the element, so you are setting the x value here to be 45% of that (so 22.5% of the element width).